src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -48,11 +52,28 @@ class ComposeUp extends DefaultTask {
         }
     }
 
+    protected void initializeComposeFilesInfo() {
+        composeFiles = extension.useComposeFiles
+        filesSpecified = composeFiles && composeFiles.size() > 0
+    }
+
+    protected void honourComposeFileOverride() {

```

Comment:
Why is this needed? According to [the documentation](https://docs.docker.com/compose/reference/overview/), it will look for `docker-compose.override.yml` as well. I would prefer to do nothing if compose files are not specified.